package.json

```diff
@@ -27,17 +30,21 @@
   },
   "devDependencies": {
     "@types/react": "18.3.2",
+    "@types/react-dom": "^18.3.5",
     "autoprefixer": "^10.0.1",
     "eslint": "^8",
     "eslint-config-next": "14.1.0",
     "eslint-config-prettier": "^9.1.0",
     "husky": "^9.0.11",
     "lint-staged": "^15.2.2",
+    "next": "15.1.0",
```

Comment:
don't unncessarily update anything. Take a pull from dev branch and remove your local package.json file